src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -59,7 +63,11 @@ const Dashboard = (props) => {
             <CardWrapper
               key={card.key}
               mb={5}
-              onSubmitButton={onGenerateProductKeyClick}
+              onSubmitButton={
+                card.action.buttonText === "EXPLORE THE DATABASE"
```

Comment:
Better to apply condition on card.key instead of text

Reply:
done